elements/pfe-datetime/src/pfe-datetime.js

```diff
@@ -86,8 +90,11 @@ class PfeDatetime extends PFElement {
         "2-digit": "2-digit"
       },
       month: {
+        numeric: "numeric",
+        "2-digit": "2-digit",
+        narrow: "narrow",
         short: "short",
-        long: "long"
+        long: "long",
```

Comment:
I think the trailing comma will need to be removed - won't this cause errors in some compilations?